web/.env.example

```diff
@@ -44,3 +44,17 @@ PUSHER_APP_CLUSTER=mt1
 
 MIX_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
 MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
+
+# Initial password for the Admin user
+ADMIN_INITIAL_PASSWORD=admin1
+
+# This var should be set to a valid locale
+DEFAULT_LANGUAGE=en
```

Comment:
I haven't decided how much should be an environment variable and how much should be stored in the new config array. They ostensibly do the same thing so we can discuss what pattern we prefer.